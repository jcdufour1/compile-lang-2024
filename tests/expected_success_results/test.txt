thing with colon
thing with colon
thing without colon
thing without colon
thing with colon no let
thing with colon no let
thing without colon no let
thing without colon no let
new programming language
hello: 89
67
bye
new programming language
455
idx_2: 0
idx_2: 1
idx_2: 2
idx_2: 3
idx_2: 4
idx_2: 5
idx_2: 6
