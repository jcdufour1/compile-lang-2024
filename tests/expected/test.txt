thing
thing
new programming language
hello: 89
67
bye
new programming language
455
